LoginPlugin/iOS/ZappLoginPluginExample-iOS/FacebookLoginPlugin.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  FacebookLoginPlugin.swift.swift
+//  ZappLoginPlugins
+//
+//  Created by Roi Kedarya on 17/05/2018.
+//  Copyright © 2018 Applicaster. All rights reserved.
+//
+
+import Foundation
+import ZappPlugins
+import ApplicasterSDK
+import FacebookCore
+import FacebookLogin
+
+@objc class FacebookLoginPlugin: NSObject, ZPLoginProviderProtocol, ZPLoginProviderUserDataProtocol {
```

Comment:
can you add a comment with a few words about what this class is doing and its role?
explain that this is the actual plugin and etc.

Reply:
@RoiKed what about this one?